mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -140,11 +150,13 @@ def list_endpoints(
     Or by using a `,` (comma) seperator:
     `api/projects/{project}/model-endpoints/?label=mylabel=1,myotherlabel=2`
     """
-
-    client = get_v3io_client()
+    secrets = get_model_endpoint_secrets(request)
+    client = get_v3io_client(
+        endpoint=secrets[V3IO_API], access_key=secrets[V3IO_ACCESS_KEY]
+    )
     cursor = client.kv.new_cursor(
```

Comment:
@Michaelliv would be better to init v3io client once, i believe there is a way to pass the user/access per request as well

Reply:
Its kind of a singleton, so it inits once for the same set of params